bundles/core/org.eclipse.smarthome.core.ephemeris/pom.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.eclipse.smarthome.bundles</groupId>
+    <artifactId>core</artifactId>
+    <version>0.10.0-SNAPSHOT</version>
+  </parent>
+  <groupId>org.eclipse.smarthome.core</groupId>
+  <artifactId>org.eclipse.smarthome.core.ephemeris</artifactId>
+
+  <packaging>eclipse-plugin</packaging>
+
+  <name>Eclipse SmartHome Core Ephemeris</name>
+
+  <properties>
+    <bundle.symbolicName>org.eclipse.smarthome.core.ephemeris</bundle.symbolicName>
+    <bundle.namespace>org.eclipse.smarthome.core.ephemeris</bundle.namespace>
+  </properties>
```

Comment:
I see that two properties in some `pom.xml` files but don't know why them has been added. Can you please give me some information?

Reply:
They have been removed in the last commit.